src/Generators/RequestsGenerator.php

```diff
@@ -110,6 +110,12 @@ protected function getSearchValidationParameters()
 
         $parameters['string-required'] = ['with.*'];
 
+        if (!empty($parameters['boolean-required'])) {
```

Comment:
such changes didn't affect for generating of create request class

<img width="592" alt="image" src="https://github.com/RonasIT/laravel-entity-generator/assets/9486872/7e11a27d-7d5b-4bc3-bfcd-57fbfdb11a93">